website/docs/reference/service-accounts.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Service accounts
+---
+
+:::info
+
+Service accounts is an upcoming enterprise feature, and is scheduled to become available in one of the next few releases.
+
+:::
+
+Service accounts enable Unleash admins to create accounts that act as users and respect the same set of permissions, however they don't represent real users. These accounts do not have a password and cannot log in to the Unleash UI. Instead, they are intended to be used to access the Unleash API programmatically, providing integrations an identity.
```

Comment:
Before we say anything about what service accounts let admins do, can we say what they are? 

Something like this?

```suggestion
Service accounts are accounts that act as normal Unleash users and that respect the same set of permissions, but that don't represent real users. These accounts do not have a password and cannot log in to the Unleash UI. Instead, they are intended to be used to access the Unleash API programmatically, providing integrations an identity.
```